its all connected: read the description 👇🏼

kathinka opened the pull request:
- Update button text in Login page to "Sign in" instead of "Sign up".
- Connected points from form, travel, points and user.
- added useContext to be able to pass values among components in dash.
- made calculation for points.
- updated travel model to be able to pass on points to user and points db.
- updated user model.
- refactored travelform so we can get updates on points on the fly so the user can see the value of each travelchoice.
- and som nip and tucks.
- added ripple effect on ego button
- fixed links not working on landingpage
- fixed pointscard so it will update every 5 seconds
- fixed styling and global style
- fixed egobtn so it its easier to use overall

kathinka commented:
DO NOT MERGE. only look don't touch. 😂 (conflict in Travelform - i restructured it)

El1an3 commented on frontend/src/pages/Login.jsx:
```diff
@@ -287,7 +287,7 @@ export const Login = () => {
             <ButtonAndSpinnerContainer>
               <EgoButton type="submit" disabled={loading}>
-                {loading ? "Loading..." : "Sign up"}
+                {loading ? "Loading..." : "Sign in"}
```
Should be Log in.

El1an3 reviewed (approved):
Looks good to me...
Just lookie, no touchie 😂

vittoriamatteoli reviewed (approved):
Also just looking 😂 Smart adjustments📱

kathinka commented:
Update button text in Login page to "Sign in" instead of "Sign up".
Connected points from form, travel, points and user.
added useContext to be able to pass values among components in dash.
made calculation for points.
updated travel model to be able to pass on points to user and points db.
updated user model.
refactored travelform so we can get updates on points on the fly so the user can see the value of each travelchoice.
and som nip and tucks.
added ripple effect on ego button
fixed links not working on landingpage
fixed pointscard so it will update every 5 seconds
fixed styling and global style
fixed egobtn so it its easier to use overall
